app/controllers/UploadController.scala

```diff
@@ -124,7 +108,7 @@ class UploadController @Inject() (
       val pageHeadingUploading = "Uploading judgment"
 
       uploadStatus match {
-        case Some(InProgressValue.value) =>
+        case Some(InProgressValue.value) | Some(CompletedWithIssuesValue.value) =>
```

Comment:
Will this still show an error on the page if the status is `CompletedWithIssuesValue`?

Reply:
Yes it will